org.mixedrealitytoolkit.spatialmanipulation/ObjectManipulator/ObjectManipulator.cs

```diff
@@ -525,12 +551,16 @@ protected struct LogicImplementation
 
         private bool ShouldSmooth => (IsGrabSelected && SmoothingNear) || (!IsGrabSelected && SmoothingFar);
 
+        private bool UseForces => rigidBody != null && !rigidBody.isKinematic;
+
         private Rigidbody rigidBody;
 
         private bool wasGravity = false;
 
         private bool wasKinematic = false;
 
+        private MovementType currentRigidbodyMovementType;
+
```

Comment:
Can we explicitly set a default value for `currentRigidbodyMovementType`?

Reply:
Done